src/__mocks__/axios.ts

```diff
@@ -0,0 +1,7 @@
+import type { AxiosStatic } from 'axios';
```

Comment:
I object to smacking __mocks__ in ./src/

Reply:
👍 I've removed this